pkg/queue/error.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2022 Outreach Corporation. All Rights Reserved.
+//
+// Description: error
```

Comment:
Same thing for the description here, it should go into detail about why this file exists.